python/tank/descriptor/io_descriptor/shotgun_entity.py

```diff
@@ -162,18 +162,10 @@ def _get_bundle_cache_path(self, bundle_cache_root):
         # make it as short as possible for hosted sites
         base_url = base_url.replace(".shotgunstudio.com", "")
 
```

Comment:
Looks like the comments above in this method need to be updated with the new path format?